index.html

```diff
@@ -1276,7 +1276,7 @@ <h3 id="lang">Translation</h3>
   <p>Alternatively you can overwrite some translations with the <code>lang</code> option. It is very useful for custom
     operators and conditions.<br>
     Default English strings are
-    <a href="https://github.com/mistic100/jQuery-QueryBuilder/search?q=en.json+in:path">
+    <a href="https://github.com/mistic100/jQuery-QueryBuilder/blob/master/src/i18n/en.json">
```

Comment:
```suggestion
    <a href="https://github.com/search?q=repo:mistic100/jQuery-QueryBuilder%20path:en.json&type=code">
```

There are multiple files, that was the purpose of the search query instead of direct link